### DISCUSSION TITLE
how to drop a table that not listed in `--drop-tables`?

### DISCUSSION THREAD

**Discussion by fontamin** [fonttools/fonttools#3166] [Q&A] [answered]
Hi
i wanted to drop `MVAR` table from my VF to create some versions to be used in one app that doesn't support MVAR properly, but i realized:
`pyftsubset myfont.ttf * --drop-tables-=MVAR`
not work because `MVAR` isn't in the `--drop-tables` table list, so how can i do that from `pyftsubset` or a fonttools based python script?

**Comment by justvanrossum** [comment, marked as answer]
I'm pretty sure you need to write

    --drop-tables+=MVAR

as you are adding to the list of tables to be dropped.

**Reply by fontamin** [reply]
thank you. it's working now.
i thought it supposed to be `-=`
